cluster/userdata-master.yaml

```diff
@@ -810,6 +813,67 @@ write_files:
               - kubectl
               - proxy
 
+  - path: /srv/kubernetes/manifests/services/kube-apiserver-readonly-svc.yaml
+    content: |
+      apiVersion: v1
+      kind: Service
+      metadata:
+        name: apiserver-svc-ro
+        namespace: kube-system
+      spec:
+        type: ClusterIP
+        ports:
+          - port: 80
+            protocol: TCP
+            targetPort: 8082
+        selector:
+          application: kube-apiserver
+          version: v1.4.5_coreos.0
```

Comment:
the other services in this file usually don't put the `version` in the selector of the service